test/controllers/SessionsControllerSpec.scala

```diff
@@ -30,7 +30,11 @@ class SessionsControllerSpec extends PlaySpecification with Results {
   private val _id: BSONObjectID = BSONObjectID.generate()
```

Comment:
Would you want to change youtubeLink and slideShareLink to youtubeURL and slideShareURL in test cases as well?